app/models/shelter.rb

```diff
@@ -1,5 +1,17 @@
 class Shelter < ApplicationRecord
   has_many :pets
-  has_many :shelter_reviews 
+  has_many :shelter_reviews, dependent: :destroy
```

Comment:
Cool! So this means when the shelter is destroyed then the reviews are also destroyed?

Reply:
Yes! Learning about this, was the best part!